### DISCUSSION TITLE
Issue: No User Moderation Options available in Lazer? (Mute/Block, Report)

### DISCUSSION THREAD

**Discussion by IONayrus** [ppy/osu#22902] [Q&A]
So as you play multi live with other players you'll always run into the toxic people who either start cyber mobbing, which i usually mute/block immediately, or in rare cases they verbally abuse which should get easily reported. (99% because you picked a map they hated)

I've sadly now noticed that there are no user options in lazer chat or multi available or they are very well hidden. Closing the game, searching for the user on the website to block or report him there cannot be intended at all right? That's why i consider this almost more an issue than a "feature" request.

But maybe i just haven't found that functionality yet, in which case i would wish for it to be more intuitively available like in stable.

**Comment by peppy** [comment]
https://github.com/ppy/osu/pull/21736 some of this is WIP.
